packages/docs/docs/fhir-basics.md

```diff
@@ -0,0 +1,200 @@
+# FHIR Basics
+
+- Medplum stores data in the FHIR standard
+- FHIR has the following advantages
+  - Interoperability
+  - Future proofing (example: charge items for devices)
+- The page will go over the basic concepts for understanding FHIR data. [Link to FHIR docs](#)
+
+## Resources
+
+A core data object in FHIR is called a [**Resource**](https://www.hl7.org/fhir/resource.html). You can think of Resources as **objects** in object oriented languages. The FHIR standard defines a set of [**Resource Types**](/category/resources), similar to **classes**, that have been built for healthcare applications. A `Resource` is composed of multiple fields, called `Elements`, each of which can be a **primitive type** (e.g. strings, numbers, dates) or a **complex type** (e.g. JSON objects)
+
+Resources can represent a broad range of healthcare ideas, from very **concrete healthcare items** (e.g. [Patient](/api/fhir/resources/patient), [Medication](/api/fhir/resources/medication), [Device](/api/fhir/resources/device)) or more **abstract concepts** (e.g. [Procedure](/api/fhir/resources/procedure), [Care Plan](/api/fhir/resources/careplan), [Encounter](/api/fhir/resources/encounter)).
+
+### Example
+
+The example below shows an example **[Patient](/api/fhir/resources/patient) resource**, represented as JSON:
+
+```json
```

Comment:
If you use type "javascript" or "ts" rather than "json", then you won't get errors for the comments.

Reply:
Bump